### ISSUE TITLE
Codacy Issues

### ISSUE THREAD

**Issue by winder** [winder/Universal-G-Code-Sender#852]
UGS uses a service called codacy for static analysis. It helps find some basic issues like unused code. There are a lot of minor things in the report which should be fixed:

https://www.codacy.com/app/winder/Universal-G-Code-Sender/dashboard?bid=5034962

**Comment by breiler** [comment]
There is always a question of what level that is good enough when it comes to code analysis. I think these are pretty good numbers and Codacy has proven it self to help us enhance it event further so I think we can close this?

![screenshot 2018-05-07 11 40 23](https://user-images.githubusercontent.com/8962024/39695590-7448b89a-51eb-11e8-9ccf-fde24a34f47a.png)

**Comment by winder** [comment]
I agree I don't think 100% codacy compliance is a feature we need to actively work towards. The main reason to leave this ticket open is for anyone who may want to contribute in the future. It provides an easy way for them to get started.
